src/main/java/cz/startnet/utils/pgdiff/parsers/CreateTriggerParser.java

```diff
@@ -108,14 +108,13 @@ public static void parse(final PgDatabase database,
         }
 
         parser.expect("EXECUTE");
-
-        if (parser.expectOptional("PROCEDURE")) {
-            trigger.setFunction(parser.getRest());
-        } else if (parser.expectOptional("FUNCTION")) {
-            trigger.setFunction(parser.getRest());
+        if (parser.expectOptional("PROCEDURE")) {           
+        } else if (parser.expectOptional("FUNCTION")) {           
```

Comment:
this looks really weird. please use something like ``if (!parser.expectOptional("PROCEDURE") && !parser.expectOptional("FUNCTION")} {...``